marlowe-runtime-web/server/Language/Marlowe/Runtime/Web/Server/OpenAPI.hs

```diff
@@ -62,6 +72,169 @@ instance ToJSON OpenApiWithEmptySecurity where
 
 type API = "openapi.json" :> Get '[JSON] OpenApiWithEmptySecurity
 
+data OpenApiLintIssue = OpenApiLintIssue
+  { trace :: Text
```

Comment:
This could be a list of semantically entailed identifiers rather than just a blob of text.

Reply:
I agree this should be more structured ideally, but do you intend to add more checks in the future? Keeping the structure loosely typed might be beneficial until we're ready to set it in stone.

Reply:
My thoughts exactly.